source/extensions/filters/network/thrift_proxy/buffer_helper.h

```diff
@@ -69,104 +69,104 @@ class BufferHelper {
    * @param offset offset into buffer to peek at
    * @return the int8_t at offset in buffer
    */
-  static int8_t peekI8(Buffer::Instance& buffer, uint64_t offset = 0);
+  static int8_t peekBEI8(Buffer::Instance& buffer, uint64_t offset = 0);
```

Comment:
I don't think the big- or little-endian appellation applies to single bytes, so probably undo this one.

But for the rest of the methods, is there a specific reason for renaming them all? For example, are you planning on moving this helper class out of the thrift extension so it can be reused elsewhere?